app/serializers/hotel_serializer.rb

```diff
@@ -1,11 +1,12 @@
 class HotelSerializer < ActiveModel::Serializer
   include Rails.application.routes.url_helpers
   attributes :id, :name, :description, :rating, :image
-  def image
-    return unless object.image.attached?
+  belongs_to :city
+  # def image
+  #   return unless object.images.attached?
 
-    {
-      url: rails_blob_url(object.image, only_path: true)
-    }
-  end
+  #   {
+  #     url: rails_blob_url(object.images, only_path: true)
+  #   }
+  # end
```

Comment:
@Tadessejemal1 please remove this one as well, it is not necessary for the project to work and we won't need it.